README.md

```diff
@@ -1,7 +1,7 @@
 # Ethereum Swarm Bee
 
 [![Go](https://github.com/ethersphere/bee/workflows/Go/badge.svg)](https://github.com/ethersphere/bee/actions)
-[![GoDoc](https://godoc.org/github.com/ethersphere/bee?status.svg)](https://godoc.org/github.com/ethersphere/bee)
+[![GoDoc](https://godoc.org/github.com/ethersphere/bee?status.svg)](https://pkg.go.dev/github.com/ethersphere/bee)
```

Comment:
The badge url is still on godoc.org. It would be good to generate a new badge completely https://pkg.go.dev/badge/.